css/style.css

```diff
@@ -0,0 +1,903 @@
+    -webkit-transition: .3s;
+    -moz-transition: .3s;
+    -ms-transition: .3s;
+    -o-transition: .3s;
+}
+
+.social-list_item:hover {
+    background-color: #6eeee4;
+    z-index: 1;
+}
+
+.fab:hover {
+    color: rgb(73, 28, 179);
+    z-index: 0;
+}
+
+.text-contact {
+    margin-left: -2px;
+}
+
+.web {
+    margin-top: 26px;
+}
+
+@media (max-width:1200px) {
+    .container {
+        max-width: 960px
+    }
+    .header_image {
+        display: none;
+    }
+    .featured-works__images img {
+        width: 300px;
+        -o-object-fit: cover;
+        object-fit: cover;
+    }
+    .small-team {
+        padding-bottom: 100px
+    }
+    .small-team__images {
+        justify-content: space-around
+    }
+    .form {
+        width: 470px
+    }
+    .contact-contact {
+        width: 470px;
+    }
+    .map iframe {
+        width: 427px
+    }
+}
+
+@media (max-width:1000px) {
+    .container {
+        max-width: 720px
+    }
+    .main-information__text {
+        width: 300px
+    }
+    .main-image__style {
+        width: 310px;
+        -o-object-fit: cover;
+        object-fit: cover
+    }
+    .first-par_text,
+    .first-par_text_second,
+    .second-par_text {
+        line-height: 22px
+    }
+    .featured-works__images {
+        margin-bottom: 100px
+    }
+    .featured-works__images img {
+        width: 350px
+    }
+    .popup__content img {
+        height: 70vh
+    }
+    .form {
+        width: 350px
+    }
+    .contact-contact {
+        width: 350px;
+    }
+    .map iframe {
+        width: 307px
+    }
+}
+
+@media (max-width:800px) {
+    .container {
+        max-width: 540px
+    }
+    .header_text__style {
+        font-size: 30px;
+        line-height: 40px
+    }
+    .main-content {
+        padding-top: 60px
+    }
+    .main-information {
+        flex-direction: column;
+        align-items: center;
+        padding-top: 40px
+    }
+    .main-information__text {
+        width: 500px
+    }
+    .main-image__style {
+        width: 500px;
+        height: 300px;
+        margin-bottom: 40px
+    }
+    .featured-works {
+        padding-top: 80px
+    }
+    .featured-works__images {
+        margin-top: 80px
+    }
+    .featured-works__images img {
+        width: 260px
+    }
+    .popup__content img {
+        height: 50vh
+    }
+    .small-team__images {
+        margin-top: 80px
+    }
+    .contact {
+        padding-top: 50px
+    }
+    .contact-wrapper {
+        flex-direction: column;
+        align-items: center
+    }
+    .contact-form {
+        width: 520px;
+        margin-bottom: 15px
+    }
+    .contact-contact {
+        width: 520px;
+        margin-bottom: 50px
+    }
+    .map iframe {
+        width: 475px
+    }
+}
+
+@media (max-width:600px) {
+    .container {
+        width: 100%;
+        padding: 0 10px
+    }
+    .header {
+        height: 530px
+    }
+    .header_text__style {
+        margin-top: 50px;
+        margin-bottom: 50px

```

Comment:
prefer to use `rem, em, %` units because we wanna have adaptive design

Reply:
Thanks 
i fixed it
```suggestion 
.header-text__title {
          margin-top: 3rem;
          margin-bottom: 3rem;
```